### PR TITLE
Add unit to timeline tool Y label; Also fix data format in Download.

### PR DESCRIPTION
**Pull request by shifucun**
image

### CODE REVIEW

File: static/js/tools/timeline/chart.tsx [hunk 1]
```diff
@@ -200,7 +200,20 @@ class Chart extends Component<ChartPropsType> {
...
     }
     // use mprop as the ylabel
-    const ylabelText = mprop.charAt(0).toUpperCase() + mprop.slice(1);
+    let ylabelText = mprop.charAt(0).toUpperCase() + mprop.slice(1);
+
+    // TODO(shifucun):
+    // Pick unit from one place and one stat var. This assumes all the stat var
+    // observation have same unit, which is mostly wrong but not strictly true.
+    // To do this accurately need a bigger change that group the stat vars at
+    // an early stage.
```
Comment (beets) [inline static/js/tools/timeline/chart.tsx:209]: in the meantime, could we list all the available units? it'll make the issue obvious (or not, depending on the severity)

**Review by beets — APPROVED** [review]
it'll be good to also use the units in the tooltip, which can be our source of truth for each observation

File: static/js/tools/timeline/chart.tsx (hunk 1 above)
Comment (shifucun) [reply static/js/tools/timeline/chart.tsx]: Done

Comment (shifucun) [reply static/js/tools/timeline/chart.tsx]: Also added unit in tooltip

**Review by chejennifer — APPROVED** [review]

**Pull request merged**